lib/src/view/tournament/tournament_screen.dart

```diff
@@ -0,0 +1,506 @@
+        appBarTitle: SizedBox.shrink(),
+        body: Center(child: CircularProgressIndicator()),
+      ),
+    };
+  }
+}
+
+class _Body extends ConsumerWidget {
+  const _Body({required this.state});
+
+  final TournamentState state;
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    return PlatformScaffold(
+      appBarTitle: _Title(state: state),
+      body: Column(
+        children: [
+          Expanded(
+            child: Padding(
+              padding: Styles.bodySectionPadding,
+              child: SingleChildScrollView(
+                child: Column(
+                  spacing: 20,
+                  crossAxisAlignment: CrossAxisAlignment.start,
+                  children: [
+                    _Verdicts(state.tournament.verdicts),
+                    if (!state.tournament.berserkable)
+                      const Text.rich(
+                        TextSpan(
+                          children: [
+                            WidgetSpan(child: Icon(LichessIcons.body_cut, size: 16)),
+                            TextSpan(
+                              // TODO l10n
+                              text: ' No Berserk allowed',
+                            ),
+                          ],
+                        ),
+                      ),
+                    _Standing(state),
+                    if (state.tournament.featuredGame != null)
+                      _FeaturedGame(state.tournament.featuredGame!),
+                  ],
+                ),
+              ),
+            ),
+          ),
+          _BottomBar(state),
+        ],
+      ),
+    );
+  }
+}
+
+class _Title extends StatelessWidget {
+  const _Title({required this.state});
+
+  final TournamentState state;
+
+  @override
+  Widget build(BuildContext context) {
+    final timeLeft = state.tournament.timeToStart ?? state.tournament.timeToFinish;
+    return Row(
+      mainAxisAlignment: MainAxisAlignment.spaceBetween,
+      children: [
+        Flexible(child: AutoSizeText(state.tournament.fullName, maxLines: 1, minFontSize: 14.0)),
+        if (timeLeft != null)
+          CountdownClockBuilder(
+            timeLeft: timeLeft,
+            active: true,
+            tickInterval: const Duration(seconds: 1),
+            builder:
+                (BuildContext context, Duration timeLeft) => Center(
+                  child: Text(
+                    '${timeLeft.toHoursMinutesSeconds()} ',
+                    style: const TextStyle(fontFeatures: [FontFeature.tabularFigures()]),
+                  ),
+                ),
+          ),
+      ],
+    );
+  }
+}
+
+class _Standing extends ConsumerWidget {
+  const _Standing(this.state);
+
+  final TournamentState state;
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    final standing = state.tournament.standing;
+    if (standing == null) {
+      return const SizedBox.shrink();
+    }
+    return Column(
+      children: [
+        ListView.builder(
```

Comment:
This should not be a list view, since we're showing only 10 items and it's not scrolling.